cephalopod/webinterface/frontend.py

```diff
@@ -31,29 +29,23 @@ def get_server_fields(server):
 
 @bp.route('/')
 @menu('index')
+@login_required
```

Comment:
If almost everything in this blueprint requires login then better check it in an `@bp.before_request` function and either whitelist the endpoints that do not require login or move them to a different blueprint (e.g. `auth`)